### DISCUSSION TITLE
Portainer pulling old release despite using latest tag

### DISCUSSION THREAD

**Discussion by clafata94** [teslamate-org/teslamate#4170] [Q&A]
Hello,

I recently installed TeslaMate through Portainer and used `ghcr.io/teslamate-org/teslamate:latest` as the image, but after install I'm on v1.28.1 - I can see from the main page that v1.30.1 is tagged as 'latest' so why would I get an old version? I tried changing the image to 'ghcr.io/teslamate-org/teslamate:buildcache-arm64-master' but that just gave an error. 

This is on Synology and I'm running DSM 7.2.1-69057 Update 3, if that might have any impact on it. Thank you in advance for any help.

**Comment by Hanzzie18** [comment]
I have exact the same issue. You must add an extra ghcr.io registry tro portainer. 
See: https://www.reddit.com/r/portainer/comments/y9ta15/add_github_registry_ghcr_to_portainer/?tl=nl

https://rehost.diberie.com/Picture/Get/f/94691
